app/forms/disk-attach.tsx

```diff
@@ -25,6 +26,7 @@ type AttachDiskProps = {
   diskNamesToExclude?: string[]
   loading?: boolean
   submitError?: ApiError | null
+  instance: Instance
```

Comment:
Requiring the instance isn't compatible with the use of this on the instance create form, but we can figure out how to deal with that.